exist-versioning-release.md

```diff
@@ -208,101 +313,169 @@ You will require a system with:
     What is the new development version for "eXist-db"? (org.exist-db:exist) 5.4.0-SNAPSHOT: :
     ```
 
-6.  Once the prepare process completes you can perform the release. This will upload Maven Artifacts to Maven
-Central (staging), Docker images to Docker Hub, and eXist-db distributions and installer to Github releases:
-    ```
-    $ mvn -Ddocker=true -Dmac-signing=true -Djarsigner.skip=false -Darguments="-Ddocker=true -Dmac-signing=true -Djarsigner.skip=false" release:perform
-    ```
+5. Test the release artefacts as much as possible on different Operating systems (Mac, Windows, Linux).  
```

Comment:
This testing is happening at the wrong time. This is far too late in the overall process.

Once you have merged any outstanding PRs for the upcoming release, and then believe that the code is stable. You can generate a SNAPSHOT release which you can share between colleagues and test. That should be done before any release process starts. Almost always some issue is found, which should then be fixed in a new PR, and then new SNAPSHOT generated, shared and tested again.

In the past Wolfgang, Dannes, and myself, would do extensive testing before any release process was started.

Reply:
Sounds very reasonable to add in this document.